deploy/olm-catalog/service-assurance-operator/0.1.0/service-assurance-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -8,12 +8,15 @@ metadata:
           "apiVersion": "infra.watch/v1alpha1",
           "kind": "ServiceAssurance",
           "metadata": {
-            "name": "saf-default"
+            "name": "cloud1"
           },
           "spec": {
-            "eventsEnabled": false,
-            "metricsEnabled": true,
-            "haEnabled": false,
+            "events": {
```

Comment:
Again this comes from the `deploy/crd` directory and should be automatically generated with the correct version of `operator-sdk` >= 13.1 and < 13.0